aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -372,7 +372,9 @@ impl From<FeatureFlag> for AptosFeatureFlag {
             FeatureFlag::PermissionedSigner => AptosFeatureFlag::PERMISSIONED_SIGNER,
             FeatureFlag::AccountAbstraction => AptosFeatureFlag::ACCOUNT_ABSTRACTION,
             FeatureFlag::BulletproofsBatchNatives => AptosFeatureFlag::BULLETPROOFS_BATCH_NATIVES,
-            FeatureFlag::DomainAccountAbstraction => AptosFeatureFlag::DOMAIN_ACCOUNT_ABSTRACTION,
+            FeatureFlag::DomainAccountAbstraction => {
+                AptosFeatureFlag::DERIVABLE_ACCOUNT_ABSTRACTION
```

Comment:
Typo? 

```
FeatureFlag::DomainAccountAbstraction => AptosFeatureFlag::DERIVABLE_ACCOUNT_ABSTRACTION
```

?

Reply:
oops didn't see that type to rename, good catch

Reply:
Oh, this was because we did a rename!

Reply:
Can we rename "domain AA" to "derivable AA" everywhere? Or are there backwards compat issues?

Reply:
updated :) I made sure all move code was good, so no backward compatibility - this is just the feature name that slipped.